### ISSUE TITLE
Docker build behind reverse proxy no MongoDb queries executed

### ISSUE THREAD

**Issue by PlkMarudny** [dbgate/dbgate#408]
I am running a Docker build of dbgate behind a reverse proxy. When trying to execute a query against MongoDB collection 'users'

```
db.users.find( { first_name: "Kathey"}, { projection: { _id: 1, email: 1, first_name: 1, last_name : 1} })
```

Only the followoing requests are executed:
![image](https://user-images.githubusercontent.com/4361591/199015386-ff859724-2918-498b-bd8e-5093955e1e44.png)

While on a local Docker setup it looks like below:

![image](https://user-images.githubusercontent.com/4361591/199015652-c9dab187-c542-4bf0-99e3-c6210fd971cb.png)

So basically after the successful 'write' POST request nothing else is executed and the query runs indefinitely.  
Maybe not exactly related to dbgate itself, but I have never seen the similar issue with this particular reverse proxy/Svelte app combination.

**Comment by janproch** [comment]
It seems like problems with event stream. There should be stream request (near the beginning of network sommunication), this stream is used for server-side-evenets (eg. notifications, when query execution is finished)

Are there any events in this stream request (try query execution for receive these events)

![image](https://user-images.githubusercontent.com/2543891/201079804-de6fc7f7-9028-4228-b5da-f1f88da1c8ba.png)

**Comment by PlkMarudny** [comment]
Wel, I am using IIS as a reverse proxy (forgot to mention) SSE will not get
through. This is kind of normal for SSE and reverse proxy, I learned this
some time ago, well, there is no easy solution (personally I was forced to
use websockets at the end).

Tomasz Plonka
